modules/api/src/main/java/org/apache/ignite/schema/SortedIndexColumn.java

```diff
@@ -22,7 +22,7 @@
  */
 public interface SortedIndexColumn extends IndexColumn {
```

Comment:
do we have any plans to support null ordering?

Reply:
Yep, I've missed this. Let's create a ticket for this.
I'm not sure it would be the only good idea to add options like ASC_NULL_FIRST to SortOrder enum, as long as non-nullable columns are supported.